src/main/java/net/snowflake/ingest/connection/RequestBuilder.java

```diff
@@ -683,7 +683,13 @@ public HttpPost generateStreamingIngestPostRequest(
     URI uri = null;
     try {
       uri =
-          new URIBuilder().setScheme(scheme).setHost(host).setPort(port).setPath(endPoint).build();
+          new URIBuilder()
+                  .setScheme(scheme)
+                  .setHost(host)
+                  .setPort(port)
+                  .setPath(endPoint)
+                  .setParameter(REQUEST_ID, UUID.randomUUID().toString())
```

Comment:
should we log this requestid?

Reply:
Valid point. added requestId in above debug log! we dont need everywhere IMO. this requestId is mainly for internal debugging.